artisoptions_christinenonthermal.h

```diff
@@ -16,11 +16,15 @@ constexpr bool FORCE_SPHERICAL_ESCAPE_SURFACE = false;
 
 constexpr int NLTEITER = 30;
 
-constexpr bool LEVEL_IS_NLTE(int element_z, int ionstage, int level) {
+constexpr int ION_NLEVELS_NLTE(int element_z, int ionstage) {
```

Comment:
This definition (which includes the ground state) doesn't match get_nlevels_excited_nlte(). I suggest either renaming to ION_NLEVELS_EXCITED_NLTE (and subtracting one from the results), or changing the definition of get_nlevels_excited_nlte() to get_nlevels_nlte() to include ground state (and subtracting one everywhere that the old function was used).